feature-profile/src/main/java/cm/aptoide/pt/profile/presentation/ProfileView.kt

```diff
@@ -258,19 +255,16 @@ private fun AptoideProductsCard(openLink: (String) -> Unit) {
         onClick = { openLink("https://aptoide-uploader.pt.aptoide.com/app") }
       )
       Row(
-        modifier = Modifier
-          .fillMaxWidth()
-          .padding(start = 16.dp, top = 6.dp),
-        horizontalArrangement = Arrangement.spacedBy((-4).dp),
+        modifier = Modifier.fillMaxWidth(),
         verticalAlignment = Alignment.CenterVertically,
       ) {
         Text(
           text = "Follow us",
-          modifier = Modifier.padding(end = 12.dp),
```

Comment:
Don't remove this end padding, It should actually be even larger to match design